components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -28,22 +30,25 @@ using namespace net::registry_controlled_domains;
 
 namespace brave_shields {
 
-bool GetDefaultFromResourceIdentifier(const std::string& resource_identifier) {
-  if (resource_identifier == "ads") {
+bool GetDefaultFromResourceIdentifier(const std::string& resource_identifier,
+    const GURL& primary_url, const GURL& secondary_url) {
+  if (resource_identifier == brave_shields::kAds) {
     return false;
-  } else if (resource_identifier == "trackers") {
+  } else if (resource_identifier == brave_shields::kTrackers) {
```

Comment:
Not sure why we didn't use these constants, but we should have been, so updated it.